inventory.yml.sample

```diff
@@ -159,6 +159,13 @@ all:
           cert_organizational_unit: Lab
           cert_state: NC
 
+          # If you wish for crucible to mirror a filtered operator catalog
+          # then uncomment the line blow. This will add a signifcant amount of time to the execution time.
+          # populate_operator_catalog: True
+          # You can run the playbooks/populate_registry.yml stand alone but you will need 
+          # to set populate_operator_catalog to True. 
+          # You can also set mirror_packages to modify the default set.
+
```

Comment:
I don't really think its an advanced feature. Its going be required by most people doing disconnected installations if you want to do anything interesting e.g. making the deployed cluster a management cluster you will need ACM or gitops operators, etc. 

The reason its off by default is because its slow and can be done later or in even parallel to the installation.

Reply:
Perhaps I could add a comment in the inventory and which links to the docs.